client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.jsx

```diff
@@ -85,10 +98,30 @@ const DataFilesProjectMembers = ({
   const columns = [
     memberColumn,
     {
-      Header: 'Access',
-      accessor: 'access',
+      Header: 'Permissions',
+      accessor: 'user.username',
+      id: 'project-role',
+      className: 'project-members__cell',
+      Cell: projectId
+        ? (el) => (
+            <ProjectRoleSelector projectId={projectId} username={el.value} />
+          )
+        : (el) => (
+            <span>{mapAccessToRoles(el.row.original.access).projectRole}</span>
+          ),
+    },
+    {
+      Header: 'Role',
+      accessor: 'user.username',
+      id: 'role',
       className: 'project-members__cell',
-      Cell: (el) => <span className={styles.access}>{el.value}</span>,
+      Cell: projectId
+        ? (el) => (
+            <SystemRoleSelector projectId={projectId} username={el.value} />
```

Comment:
Perhaps we should disable these selectors while either "Transfer Ownership" is active, or while actions are taking place, i.e. `isMutating`